<launch>
  <include file="$(find cpl_collab_manip)/launch/load_urdf.launch" />
  <!--
  <include file="$(find cpl_collab_manip)/launch/load_sensors.launch" />
  <param name="/rate" value="125" />
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" ></node>
  -->

  <node name="ur_sim_sim1" pkg="ur_cart_move" type="ur_arm_sim.py"  output="screen">
    <param name="use_gui" value="false"/>
    <param name="prefix" value="/sim1"/>
  </node>
  <node name="sim1_state_publisher" pkg="robot_state_publisher" type="state_publisher">
    <remap from="/joint_states" to="/sim1/joint_states" />
    <param name="tf_prefix" value="/sim1" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="sim1_static_tf" args="0 0 0 0 0 0 /base_link /sim1/base_link 100" />
  <include file="$(find ur_cart_move)/launch/spline_traj_as.launch">
    <arg name="arm_prefix" value="/sim1" />
  </include>


  <node name="ur_sim_sim2" pkg="ur_cart_move" type="ur_arm_sim.py" output="screen">
    <param name="use_gui" value="false"/>
    <param name="prefix" value="/sim2"/>
  </node>
  <node name="sim2_state_publisher" pkg="robot_state_publisher" type="state_publisher">
    <remap from="/joint_states" to="/sim2/joint_states" />
j   <param name="tf_prefix" value="/sim2" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="sim2_static_tf" args="0 0 0 0 0 0 /base_link /sim2/base_link 100" />


  <node name="ur_sim_sim3" pkg="ur_cart_move" type="ur_arm_sim.py" output="screen">
    <param name="use_gui" value="false"/>
    <param name="prefix" value="/sim3"/>
  </node>
  <node name="sim3_state_publisher" pkg="robot_state_publisher" type="state_publisher">
    <remap from="/joint_states" to="/sim3/joint_states" />
j   <param name="tf_prefix" value="/sim3" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="sim3_static_tf" args="0 0 0 0 0 0 /base_link /sim3/base_link 100" />
</launch>
